### DISCUSSION TITLE
list of links instead of a dropdown

### DISCUSSION THREAD

**Discussion by mflpopescu** [carltongibson/django-filter#1495] [Q&A] [answered]
I'm using django-filter's **ChoiceFilter** from django-filter which generates a dropdown to filter results and it works great.

However a dropdown is not what I have in mind for the design, instead I would like to use a list of links, but I can't find if there is such a filter.

How can I have a l have a list of links instead of a dropdown ?

**Current:**
![qb9MI](https://user-images.githubusercontent.com/22353083/164732581-0378d1fc-aa04-42ea-986f-b8d766c602b5.png)
**Desired:**
![DhvnN](https://user-images.githubusercontent.com/22353083/164732668-cbe76d12-ddf4-4aec-9dc8-ee396e819899.png)

***filters.py***
```
class manufacturerFilter(django_filters.FilterSet):

    manufacturer=django_filters.ChoiceFilter(choices=[])

    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.filters['manufacturer'].extra['choices'] = [
            (wheel, wheel)
            for wheel in WheelItem.objects.values_list('manufacturer', flat=True).distinct()
        ]
```

**Comment by carltongibson** [comment, marked as answer]
Did you checkout `LinkWidget`? https://django-filter.readthedocs.io/en/stable/ref/widgets.html#linkwidget

**Reply by mflpopescu** [reply]
Thanks for the pointer. 
That's all that needed to be added.
```    
manufacturer=django_filters.ChoiceFilter(choices=[],widget = django_filters.widgets.LinkWidget)
```
